attractions/package.json

```diff
@@ -2,12 +2,22 @@
   "name": "attractions",
   "version": "2.2.4",
   "description": "A UI kit for Svelte",
+  "homepage": "https://illright.github.io/attractions/",
+  "bugs": "https://github.com/illright/attractions/issues",
   "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/illright/attractions.git",
+    "directory": "attractions"
+  },
   "svelte": "index.js",
   "module": "dist/index.mjs",
   "main": "dist/index.js",
+  "unpkg": "dist/custom-elements.js",
+  "sideEffects": false,
```

Comment:
I think the `custom-elements.js` counts as a side effect, right?

Reply:
Hm. Well, that's true, but then again, `custom-elements.js` is not meant to be imported... But then again, `sideEffects` accepts an array of files that have side effects. Yeah, I think we should add it

Reply:
It is meant to be imported when not using Svelte or a library that is somehow compatible with Svelte components.

Reply:
Not imported, simply included, since it's an IIFE

Reply:
But it still exposes a global variable, `attractions`, on which you can find all components and utils